src/pykeen/models/unimodal/kg2e.py

```diff
@@ -73,21 +73,26 @@ def __init__(
         r"""Initialize KG2E.
 
         :param embedding_dim: The entity embedding dimension $d$. Is usually $d \in [50, 350]$.
-        :param dist_similarity: Either 'KL' for kullback-liebler or 'EL' for expected liklihood. Defaults to KL.
+        :param dist_similarity: Either 'KL' for Kullback-Liebler or 'EL' for expected likelihood. Defaults to KL.
```

Comment:
It's actually [Kullback LEIBLER](https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence)